### PR TITLE
avatar

### PR DESCRIPTION
**Pull request by anupkgurung**

### CODE REVIEW

File: avatar.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+:root {
```
Comment (Naveenchandar) [inline avatar.css:1]: Any specific reason for declaring variables ? Because, Variables haven't used.

File: avatar.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    --white : rgb(255,255,255,1);
+    --gray : rgb(128,128,128,1);
+}
+.avatar-container {
+    display: flex;
+}
+.items {
+    margin: 2rem;
```
Comment (Naveenchandar) [inline avatar.css:9]: Suggestion: 
**.avatar** class will override **.items**  class. So, I guess you can remove this style and if **.items**  class is not used, you can remove it from class attribute too.

File: avatar.css (hunk 1 above)
Comment (anupkgurung) [reply avatar.css:1]: no the root i used just to check it's functioning with badges root and few things will be moved to utils or main css.

File: avatar.css (hunk 2 above)
Comment (anupkgurung) [reply avatar.css:9]: items class i used for documentation i just forgot to remove it before committing here. I had placed all items inside a container.

**Review by Naveenchandar — APPROVED** [review]
Looks good.

**Pull request closed without merging**